lms/djangoapps/teams/static/teams/templates/create-team-button.underscore

```diff
@@ -0,0 +1,4 @@
+<div class="add-team">
+  <h3 class="title">Having trouble finding a team to join?</h3>
+  <p class="copy">Try <a href="">browsing all teams</a>, <a href="">searching team descriptions</a>, or if you still can't find one, you can <a class="create-team" href="">create a new team</a>.</p>
```

Comment:
Rewording. +1 to @frrrances' suggestion about adding context.
"Try browsing all teams or searching team descriptions. If you still can't find a team to join, create a new team within this topic."